jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc

```diff
@@ -23,58 +23,57 @@ toc::[]
 the health of your system and debug issues with your installation. However, requirements differ vastly between use
-cases. Hence, features around local observability are provided on an as-is basis and we kindly ask to make use of the
-flexibility and extensibility of the system should you feel that the functionality provided is not sufficient for your
-needs. Most importantly, note that
+cases. Hence, features around local observability are provided on an as-is basis, and we kindly ask to make use of the
+flexibility and extensibility of the system, should you feel that the functionality provided is not sufficient for your
+needs.
 
-* You cannot modify dashboards that are provided by CircleCI. However, you can make copies of them that are editable.
-* Data is retained for a maximum of 15 days
+Most importantly, take note of the following:
+
+NOTE: You cannot modify dashboards that are provided by CircleCI. However, you can make copies of them that are editable.
+NOTE: Data is retained for a maximum of 15 days.
 
 === Grafana
 Server ships with https://grafana.com/[Grafana] which has become the world's most popular technology used to compose
-observability dashboards. We have integrated with Prometheus & Loki Logs so you can create dashboards, set up alerts,
-search for errors and more.
+observability dashboards. We have integrated with https://prometheus.io/[Prometheus] & https://grafana.com/oss/loki/[Loki] logging,
+so you can create dashboards, set up alerts, search for errors, and more.
 
 === Loki
-Loki is a horizontally-scalable, highly-available, log aggregation system inspired by Prometheus. It is designed to be
-very cost effective and easy to operate. It does not index the contents of the logs, but rather a set of labels for each
+https://grafana.com/oss/loki/[Loki] is a horizontally-scalable, highly-available, log aggregation system inspired by https://prometheus.io/[Prometheus].
+It is designed to be very cost effective and easy to operate. It does not index the contents of the logs, but rather a set of labels for each
 log stream.
 
 === Prometheus
-https://prometheus.io/[Prometheus] is a leading monitoring and alerting system for Kubernetes. Server ships with basic
+https://prometheus.io/[Prometheus] is a leading monitoring and alerting system for Kubernetes. Server 3.x ships with basic
 implementation of monitoring common performance metrics.
 
 === Telegraf
 Most services running on server will report StatsD metrics to the https://www.influxdata.com/time-series-platform/telegraf/[Telegraf] pod running in server.
 The configuration is fully customizable, so you can forward your metrics from Telegraf to any output that is supported
-by Telegraf via https://docs.influxdata.com/telegraf/v1.17/plugins/[output plugin]. By default, it will provide a
+by Telegraf via the https://docs.influxdata.com/telegraf/v1.17/plugins/[output plugin]. By default, it will provide a
```

Comment:
Actually, its a range of output plugins that's available

Reply:
@robinsturmeit -- changed to `via https://docs.influxdata.com/telegraf/v1.17/plugins/#output-plugins[output plugins]`

Better?